dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -0,0 +1,73 @@
+class LtiV1Controller < ApplicationController
+  # Don't require an authenticity token because LTI Platforms POST to this controller.
+  skip_before_action :verify_authenticity_token
+
+  # [GET/POST] /lti/v1/login(/:platform_id)
+  def login
+    if params[:client_id]
+      query_params = {client_id: params[:client_id], issuer: params[:iss]}
+    elsif params[:platform_id]
+      query_params = {platform_id: params[:platform_id]}
+    else
+      return unauthorized_status
+    end
+
+    begin
+      lti_integration = LtiIntegration.find_by!(query_params)
+    rescue ActiveRecord::RecordNotFound
+      return unauthorized_status
+    end
+
+    auth_request_body = {
+      scope: "openid",
```

Comment:
https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#ruby-string-quotes

Reply:
Ah, yes I've seen single quotes and double quotes in the project, will change to single to stay standardized.